spec/02-integration/000-new-dao/01-db_spec.lua

```diff
@@ -50,10 +51,33 @@ for _, strategy in helpers.each_strategy() do
           error("unknown database")
         end
       end)
+
+      if strategy == "postgres" then
+        it("initializes infos with custom schema", function()
+          local conf = utils.table_merge(helpers.test_conf)
```

Comment:
Did you mean to use `deep_copy()` maybe? The intent would be clearer (ditto in similar places in this PR).

Reply:
Changed to `deep_copy`.